nixops/nix_expr.py

```diff
@@ -1,55 +1,64 @@
 import functools
 import re
 import string
-from typing import Optional
+from typing import Any, Dict, List, Optional, Sequence, Tuple, TypeVar, Union, overload
 from textwrap import dedent
 
+import sys
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
I think this is the source of the build issue. I'm not yet sure how to fix it, though.